translator/internaldata/resource_to_oc.go

```diff
@@ -106,17 +107,48 @@ func internalResourceToOC(resource pdata.Resource) (*occommon.Node, *ocresource.
 	return &ocNode, &ocResource
 }
 
-func attributeValueToString(attr pdata.AttributeValue) string {
+func attributeValueToString(attr pdata.AttributeValue, jsonLike bool) string {
```

Comment:
do we even translate this back from string?

Reply:
I am not sure I understand the question.